dashboard/client/dashboard.js

```diff
@@ -5,8 +5,74 @@ Template.dashboard.onCreated(function () {
   // Get reference to template instance
   const templateInstance = this;
 
-  // Create reactive variable for Elasticsearch host
+  // Create reactive variable for Elasticsearch host & data
   templateInstance.elasticsearchHost = new ReactiveVar();
+  templateInstance.elasticsearchData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        // Create data range with interval
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+          },
+          aggs: {
+            // Get average value of response time for each range
+            avg_response_time: {
+              avg: {
```

Comment:
If I just say what to fix, and you just fix it, you won't maybe understand why it was worth fixing.

It is important that we understand the following concepts:

- mean
- median
- percentiles
- skewness

It may not be worth fixing for this specific case, because of performance trade-offs you already mention.